prototype/app/views/how-it-works.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layouts/main.html" %}
+
+{% set pageTitle = "How it works" %}
+
+{% block content %}
+
+<div class="govuk-width-container ">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">How it works</h1>
+      <p>How the AI finds the themes and summarises them:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>Responses to the consultation are organised by similar themes for each question.</li>
+        <li>These themes generate a list of consultation themes.</li>
+        <li>Individual lists of themes are combined into one list for the entire consultation.</li>
+        <li>The LLM checks each response to see which theme it contains.</li>
+        <li>The LLM generages a summary of the responses’ themes.</li>
```

Comment:
Typo - "generages" -> "generates".

Reply:
Thanks - fixed now. I'll squash all these little commits during the rebase!